samcli/commands/sync/command.py

```diff
@@ -144,6 +144,7 @@
 @capabilities_option(default=DEFAULT_CAPABILITIES)  # pylint: disable=E1120
 @experimental
 @pass_context
+@track_long_event("SyncUsed", "Start", "SyncUsed", "End")
```

Comment:
are we able to measure the duration of each sync flow if we use `track_long_event` here? My understanding is this would track the entire `sam sync` command run.

Reply:
This would create two Events, `("SyncUsed", "Start")` and `("SyncUsed", "End")`. Since both Events are given timestamps at the time of their creation, we would be able to interpolate the duration of the command by taking the difference of the two times.

Reply:
We also get the same measurement from `track_command` if I didn't misunderstand anything.
If we do it this way, can we get individual duration for each sync flow?

Reply:
We could, though with this current implementation, we wouldn't be able to tell which pairs of events would match with which since there's the possibility that they share a thread with the main function (unless I'm mistaken). I could add new Events `SyncFlowStart` and `SyncFlowEnd` which would have the same values as `SyncFlow` to measure duration and allow us to see the types of flows and their duration, which may allow us to collect that information.